gui/src/components/Taipy/Chart.tsx

```diff
@@ -336,8 +337,11 @@ const Chart = (props: ChartProp) => {
     const skelStyle = useMemo(() => ({ ...style, minHeight: "7em" }), [style]);
 
     const dataPl = useMemo(() => {
-        const datum = data && data[dataKey];
-        return datum
+        if (data === null) {
```

Comment:
this gives me an error if I don't set the test as
 data === null && lastDataPl.current

Reply:
Whao